tracer/src/Datadog.Trace/IAST/Dataflow/AspectClassAttribute.cs

```diff
@@ -9,59 +9,58 @@
 using System.Linq;
 using Datadog.Trace.Iast.Helpers;
 
-namespace Datadog.Trace.Iast.Dataflow
+namespace Datadog.Trace.Iast.Dataflow;
+
+[AttributeUsage(AttributeTargets.Class)]
+internal class AspectClassAttribute : Attribute
```

Comment:
nit: we should probably add `sealed` to all these if possible

Reply:
Fixed